src/httpserver/handlers/redirect.clj

```diff
@@ -0,0 +1,4 @@
+(ns httpserver.handlers.redirect)
+
+(defn call [request]
+  {:code 302 :headers {"Location" (str "http://" (get (:headers request) "Host") "/")} :body ""})
```

Comment:
What happens if the `Host` header is not present? I'm assuming this is what cob-spec is providing, but maybe consider a fallback case?